tap_mongodb/sync_strategies/change_streams.py

```diff
@@ -12,97 +13,105 @@
 
 SDC_DELETED_AT = "_sdc_deleted_at"
 RESUME_TOKEN_KEY = 'token'
-MAX_AWAIT_TIME_MS = 120000  # 2 minutes
+MAX_AWAIT_TIME_MS = 300000  # 5 minutes
```

Comment:
is this how long it will wait for new events? And if no new events then it will exit?
If yes, then what happens if every minute we have 1 new event, will it run for very long time?

Reply:
Yes, if there are no event after 5min it will exit. Highly active collections that use log based will cause the tap to run for a long time and possibly never stopping unless we set a small timeout.

There is a risk with mixing log based and other methods but given that we don't plan on using incremental nor this tap's full table then I think we're safe ... for now.